src/components/Header/components/LowerNav.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { chakra, useMultiStyleConfig, HStack } from "@chakra-ui/react";
+
+import Button from "../../Button/Button";
+import Icon from "../../Icons/Icon";
+import Link from "../../Link/Link";
+import List from "../../List/List";
+import TextInput from "../../TextInput/TextInput";
+
+const HeaderLowerNav = chakra(() => {
+  const styles = useMultiStyleConfig("HeaderLowerNav", {});
+
+  const [searchOpen] = useState<boolean>(false);
+
+  return (
+    <List
```

Comment:
Navigations should be wrapped in a `nav` element. When you do so, add `aria-label="Main Navigation"`. This is the aria label on the current header lower nav.